src/test/java/com/google/devtools/build/lib/runtime/BuildSummaryStatsModuleTest.java

```diff
@@ -0,0 +1,168 @@
+  private ActionKeyContext actionKeyContextMock;
+  private Class tBSSM;
+  private boolean Comma = false;
+
+  @Before
+  public void setUp() throws Exception {
+    CommandEnvironment env = mock(CommandEnvironment.class);
+    actionKeyContextMock = mock(ActionKeyContext.class);
+    reporterMock = mock(Reporter.class);
+    SkyframeExecutor skyframeExecutorMock = mock(SkyframeExecutor.class);
+    EventBus eventBusMock = mock(EventBus.class);
+    when(env.getReporter()).thenReturn(reporterMock);
+    when(env.getSkyframeExecutor()).thenReturn(skyframeExecutorMock);
+    when(skyframeExecutorMock.getActionKeyContext()).thenReturn(actionKeyContextMock);
+    when(env.getEventBus()).thenReturn(eventBusMock);
+    buildSummaryStatsModule = new BuildSummaryStatsModule();
+    buildSummaryStatsModule.beforeCommand(env);
+    /* With reflection mechanism, set some fake values to get CPU info. */
+    tBSSM = buildSummaryStatsModule.getClass();
+    Field field0 = tBSSM.getDeclaredField("statsSummary");
+    field0.setAccessible(true);
+    field0.setBoolean(buildSummaryStatsModule, true);
+    DecimalFormatSymbols dfs = new DecimalFormatSymbols();
+    if(dfs.getDecimalSeparator() == ','){
+      Comma = true;
+    }
+    else{
+      Comma = false;
+    }
+  }
+
+  private ActionResultReceivedEvent createActionEvent(Duration userTime, Duration systemTime) {
+    ActionResult result = mock(ActionResult.class);
+
+    if(userTime == null){
+      when(result.cumulativeCommandExecutionUserTime()).thenReturn(Optional.empty());
+    }
+    else{
+      when(result.cumulativeCommandExecutionUserTime()).thenReturn(Optional.of(userTime));
+    }
+
+    if(systemTime == null){
+      when(result.cumulativeCommandExecutionSystemTime()).thenReturn(Optional.empty());
+    }
+    else{
+      when(result.cumulativeCommandExecutionSystemTime()).thenReturn(Optional.of(systemTime));
+    }
+
+    when(result.spawnResults()).thenReturn(ImmutableList.of());
+    return new ActionResultReceivedEvent(null, result);
+  }
+
+  private BuildCompleteEvent createBuildEvent() {
+    BuildResult buildResult = new BuildResult(1000);
+    buildResult.setStopTime(2000);
+    return new BuildCompleteEvent(buildResult);
+  }
+
+  @Test
+  public void allCpuTimesAreSummarized() throws Exception {
+    ActionResultReceivedEvent action1 = createActionEvent(Duration.ofSeconds(50), Duration.ofSeconds(20));
+    ActionResultReceivedEvent action2 = createActionEvent(Duration.ofSeconds(5), Duration.ofSeconds(2));
+    buildSummaryStatsModule.actionResultReceived(action1);
+    buildSummaryStatsModule.actionResultReceived(action2);
+    Field field1 = tBSSM.getDeclaredField("cpuTimeForBazelJvm");
+    field1.setAccessible(true);
+    field1.setLong(buildSummaryStatsModule, 11000);
+    buildSummaryStatsModule.buildComplete(createBuildEvent());
+    if(Comma) {
+      verify(reporterMock).handle(Event.info("CPU time 88,00s (user 55,00s, system 22,00s, bazel 11,00s)"));
```

Comment:
I think the comma problem would be easier to understand if you use `String.format` here instead of having the `Comma` variable. A note on why it's done that way would be good, though.

Reply:
For this comment, I think you want it to be changed like:
verify(reporterMock).handle(Event.info(**String.format("CPU time %.2fs (user %.2fs, system %.2fs, bazel %.2fs)",88.00, 55.00, 22.00, 11.00))**);
Is it right?

Reply:
Yes, that should do it. Plus a comment on why it's not just a fixed string.

Reply:
I hope, in this way, it could be determined by the system to use '.' or ','  to expresss the floating number.